docs/getting-started/edfi-exchange/technology/quick-deploy-of-ed-fi-ods-api-development-server-using-on-premises-server.md

```diff
@@ -0,0 +1,22 @@
+# Quick Deploy of Ed-Fi ODS / API Development Server Using On Premises Server
+
+![MSDF Logo](https://edfidocs.blob.core.windows.net/$web/img/edfi-exchange/technology/msdflogo.png)
```

Comment:
We need to shrink that image. Way too large and looks bad on my screen.

![image](https://github.com/user-attachments/assets/30480011-5817-4f08-9fe7-84cb7b51aaa9)